src/api/models/token.js

```diff
@@ -1,7 +1,7 @@
 const token = (sequelize, DataTypes) => {
```

Comment:
@nkpremices I don't think we do still need this token model as for now on we are going to use Redis.

Reply:
you are the one to do it @MCFrank16